docs/semgrep-assistant/getting-started.md

```diff
@@ -148,3 +148,22 @@ There is a cap of 250 Assistant runs per month using the **Analyze** button. Ass
 ## View recommendations
 
 You can [view all of Semgrep Assistant's recommendations](/semgrep-code/findings/#filter-findings) by going to Semgrep Cloud Platform's **Findings** page and filtering by **Recommendation** or **Component**.
+
+## Write custom rules (beta)
+
+Semgrep Assistant can help you write custom rules to find issues specific to your codebase.
+
+To do so:
+
+1. Sign in to Semgrep Cloud Platform.
+2. Navigate to **Rules** > **Editor**.
```

Comment:
We're allowed to **bold** angle brackets : https://developers.google.com/style/procedures